community/kernel/src/test/java/org/neo4j/test/ProcessStreamHandler.java

```diff
@@ -54,20 +60,66 @@ public ProcessStreamHandler( Process process, boolean quiet )
         this( process, quiet, "", quiet ? IGNORE_FAILURES : PRINT_FAILURES );
     }
 
-    public ProcessStreamHandler( Process process, boolean quiet, String prefix,
-            StreamExceptionHandler failureHandler )
+    public ProcessStreamHandler( Process process, boolean quiet, String prefix, StreamExceptionHandler failureHandler )
     {
         this( process, quiet, prefix, failureHandler, System.out, System.err );
     }
 
-    public ProcessStreamHandler( Process process, boolean quiet, String prefix,
-            StreamExceptionHandler failureHandler, PrintStream out, PrintStream err )
+    public ProcessStreamHandler( Process process, boolean quiet, String prefix, StreamExceptionHandler failureHandler, PrintStream out, PrintStream err )
     {
         this.process = process;
+
+        storedOutput = new PrintStream( byteArrayOutputStream );
+        storedError = new PrintStream( byteArrayErrorStream );
+
```

Comment:
I don't think you need this. See comment below about getting the stream from the process.

Reply:
Yes, fixed that the way you suggested, thanks